### DISCUSSION TITLE
Github Student Verification failed for record 56 times

### DISCUSSION THREAD

**Discussion by Nihith-Reddy** [community/community#52741] [GitHub Education]
My school doesn't issue email id for students so I had to use my student id but it is not dated, so to supplement my student id I have dated class schedule for the semester and class syllabus for the semester.

Unfortunately when I upload all the proofs it says uploaded images are not trustworthy. From next time I applied through mobile and took photos through back camera and after applying it says Unfortunately we are not able to verify your student verification with out much information.

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 13, 2023 for [@Nihith-Reddy](https://github.com/Nihith-Reddy).

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

I already changed my username , added a bio, logged out and logged in back before applying with no success. Honestly, at this point I'm exhausted.

I created a support ticket but it has been closed because apparently github stopped providing support for student verification queries.

Can any moderator please look at this and resolve this asap, I don't know what is wrong.

Last year it was hassle free and I don't know what happened this year.

PS: All my documents are in english only.

**Comment by Nihith-Reddy** [comment]
If student verification cannot be done automatically then atleast in this case it should be manually reviewed by a human.

**Comment by Nihith-Reddy** [comment]
Hey @samus-aran can you please look into this?

**Comment by Nihith-Reddy** [comment]
Moderators can any one please help me?

**Comment by SquarishRectangle** [comment]
I am having the EXACT same issue! Github really needs to fix this.

**Comment by RayBytes** [comment]
same, also why the heck does it require you to use your real name in your github account? thats just weird, i know many people who dont want random strangers on the internet to know their real name

**Reply by Nihith-Reddy** [reply]
Yeah true

**Reply by Ljzd-PRO** [reply]
agreed

**Reply by illyrius666** [reply]
+1

**Reply by MRLIGHT02** [reply]
me to bro.....

**Reply by GhostOf0days** [reply]
Your Github account is supposed to be something that recruiters see, especially if you're a student. That differs from non-university programmers.

**Reply by MRLIGHT02** [reply]
Hey can someone guide on api

**Comment by ghost** [comment]
I also have the exact same problem but i submitted a application yesterday and it's pending at least not "rejected". Btw that was my 8th application......

**Reply by Nihith-Reddy** [reply]
There is no support from moderators as well

**Comment by AliAmmarNaseem** [comment]
Same bro, I also tried 40 times, Still getting same error as yours. Really Annoying!!

**Reply by Nihith-Reddy** [reply]
There is no support from moderators as well

**Comment by siliconsniffer** [comment]


**Reply by Nihith-Reddy** [reply]
There is no support from moderators as well

**Comment by samus-aran** [comment]
Hi @Nihith-Reddy 👋 

Must of missed your discussion so apologies on that!

Let's see how we can assist ⭐ 

You say you have a Student ID card but it's not dated. Then I suggest that you combine multiple pieces of evidence in the same photo that includes a date that your Student ID card is missing e.g. A Student ID card on top of your transcript. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. Considering your multiple pieces of evidence still select Student ID as it's the highest of the preferred.

The application can also detect differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process. Do you use a variation of your name or a nickname? Does your evidence/profile info look like it has a space between names or school names? It’s these small but important points to consider when applying that can make a much smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Hope this helps! Let us know how you get along 👍

**Reply by MidKnightXI** [reply]
I guess that you're checking also IP adresses to confirm Github edu pack, just a speculation I don't know about that but I don't see how my account would get denied from Github edu the only year where I'm studying in an other country but still with the same university.

I didn't change anything to my account since 3 years related to my identity and never got any problem except this year.

All of this process is just a pain and seeing how Github is just denying tickets from the support for Github edu just confirms it.

Github, please fix your s**** (it's the word stuff not the other one).

**Reply by Totalgeek9224** [reply]
Seconded. Also having extreme issues verifying, but the kicker is I'm even uploading a document with my name, date, school letterhead; EVERYTHING. How there is no alternative to automatic approval is astounding to me. 
@samus-aran , I'm sorry to pester you but you seem to be informed on the matter. Why is this the current state for Github students?

**Comment by FSWei** [comment]
Me too, I can still succeed in February this year. A few days ago, I was suddenly asked to re verify, but I couldn't succeed. I have tried more than 40 times!!
